e2e/tests/functional/forms.e2e.spec.js

```diff
@@ -91,6 +93,41 @@ test.describe('Persistence operations @addInit', () => {
     });
 });
 
+test.describe('Persistence operations @couchdb', () => {
+    test.use({ failOnConsoleError: false });
```

Comment:
what's generating this error?

Reply:
The 404 error from CouchDB on initial `mine` folder creation

Reply:
Do we have a bug for that filed? Seems like we're limiting some of the utility of the tests because of it

Reply:
I thought we did, but cannot find it? It's the same 404 error that is causing this error message when starting with a fresh couchDB instance:

![image](https://user-images.githubusercontent.com/9259993/197903008-9f37494e-f938-43ac-8847-58fe1f4db137.png)

We should handle this specific case gracefully somehow. I'll open an issue

Reply:
https://github.com/nasa/openmct/issues/5914